charts/kms-issuer/templates/config.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "kms-issuer.fullname" . }}-manager-config
+  namespace: {{ .Release.Namespace }}
+data:
+  controller_manager_config.yaml: |
```

Comment:
This file doesn't use the `.Values.args.localAwsEndpoint`, etc.
Do we even need it if we pass flags rather than use config? _I can't remember the "new" config file thing in kubebuilder v3_